docs/userguide/networking/overlay-security-model.md

```diff
@@ -0,0 +1,62 @@
+<!--[metadata]>
++++
+title = "Docker 1.12 overlay network security model"
+description = "Docker 1.12 Overlay Network Security Model"
+keywords = ["network, docker, documentation, user guide, multihost, swarm mode", "overlay"]
+[menu.main]
+parent = "smn_networking"
+weight=-2
++++
+<![end-metadata]-->
+
+## Note on Docker 1.12 Overlay Network Security Model
```

Comment:
I'm fine with it being here, and have it linked from the `/security/` section somewhere.